libraries/ms-common/test/test_event_queue.c

```diff
@@ -23,11 +24,11 @@ void test_event_queue_raise(void) {
   TEST_ASSERT_EQUAL(STATUS_CODE_RESOURCE_EXHAUSTED, prv_raise_event(0));
 
   Event e;
-  uint16_t i = 1;  // Start at 1 since the insertion of 0 should've failed
+  uint16_t i = EVENT_QUEUE_SIZE;
```

Comment:
Why is this change necessary? Since it's now a FIFO, we should probably change the test to verify that events are processed in order :)

Reply:
Because before you enqueued tests in reverse order and now they need to be extracted in that order. That's exactly what this change allows.

Reply:
Ah, I think an explicit array comparison might be better?

Reply:
Done. Although, the EventQueue object is private/static and the `event_queue` API only allows access to one element at a time so I sort of did this but I'm not convinced that the change was worthwhile.